### DISCUSSION TITLE
Document installation parameters

### DISCUSSION THREAD

**Discussion by God-damnit-all** [sandboxie-plus/Sandboxie#1757] [Documents and notes]
I'm guessing this might already be possible, but I couldn't find any documentation on it.

I want to run the Sandboxie Plus setup file silently, using the 'extract all files to a directory for portable use' option.

**Comment by mpheath** [comment]
Some information I mentioned here about portable silent command and logging: https://github.com/sandboxie-plus/Sandboxie/issues/1013#issuecomment-895380120

Example:

Description|Command
-|-
Silent install as portable|`setup /silent /portable=1 /currentuser /log=setup.log`
Show help messagebox|`setup /?`

`setup` represents the name of the executable.

`/verysilent` can be used instead of `/silent` to hide the progress window.

### Inno Setup documentation:

 * [Setup Command Line Parameters](https://jrsoftware.org/ishelp/topic_setupcmdline.htm)
 * [Uninstaller Command Line Parameters](https://jrsoftware.org/ishelp/topic_uninstcmdline.htm)
 * [Setup Exit Codes](https://jrsoftware.org/ishelp/topic_setupexitcodes.htm)
 * [Uninstaller Exit Codes](https://jrsoftware.org/ishelp/topic_uninstexitcodes.htm)

**Comment by God-damnit-all** [comment]
Thank you. I changed the title of my issue, but feel free to close it if needed.
